seq2seq/trainer/supervised_trainer.py

```diff
@@ -62,9 +62,12 @@ def __init__(self, expt_dir='experiment', loss=[NLLLoss()], loss_weights=None, m
     def _train_batch(self, input_variable, input_lengths, target_variable, model, teacher_forcing_ratio):
         loss = self.loss
 
+        # If attentive guidance is provided in the data set, add this as kwarg
+        kwargs = {} if 'provided_attention' not in target_variable else {'provided_attention': target_variable['provided_attention']}
```

Comment:
similar to before, can we not just pass target_variables which contains already all correct informations and do without the kwargs?